src/store/modules/ui.store.js

```diff
@@ -228,6 +233,26 @@ export default {
         noFeatureInfo(state) {
             return state.featureInfoPosition === FeatureInfoPositions.NONE
         },
+        /**
+         * Flag to display to display give feedback button/form. On localhost, it will always shown
+         * for testing purpose
+         */
+        hasGiveFeedbackButton(state, getters) {
+            if (getters.hasDevSiteWarning) {
+                return true
+            }
```

Comment:
Make no sense to have this if, if we want the feedback button on all dev site, we should add the host name in the list below, just keep one way of configuring the presence of this button.
That said on sys-map.dev.bgdi.ch make sense to have both button for testing purpose, while on sys-map.int|prod.bgdi.ch and map.geo.admin.ch we should only have the report a problem button

Reply:
Ok, to sum up:
```js
export const GIVE_FEEDBACK_HOSTNAMES = ['localhost', 'sys-map.dev.bgdi.ch', 'test.map.geo.admin.ch']

export const REPORT_PROBLEM_HOSTNAMES = [
    'localhost',
    'sys-map.dev.bgdi.ch',
    'sys-map.int.bgdi.ch',
    'sys-map.prod.bgdi.ch',
    'map.geo.admin.ch',
]
```

*adding the `test.map.geo.admin.ch'` from the JIRA ticket.